src/modules/tesla-android/filesystem/etc/network/interfaces

```diff
@@ -8,10 +8,14 @@ auto wlan0
 allow-hotplug wlan0
 iface wlan0 inet manual
 
+auto eth1
```

Comment:
This should not be needed given that usb sticks with the same interface get picked up by default

Reply:
Will remove and test.